content/doc/book/managing/_chapter.yml

```diff
@@ -13,3 +13,4 @@ sections:
   - users
   - ui-themes
   - user-content
+  - built-in-node-migration
```

Comment:
Deliberately unlinked as it's irrelevant to anyone not upgrade and not seeing the monitor.

Reply:
> Deliberately unlinked as it's irrelevant to anyone not upgrade and not seeing the monitor

Thanks.  Removed from the pull request by reverting the change.